src/locales/en.json

```diff
@@ -126,11 +126,11 @@
     "risk_tolerance_point1_percent_label": "High Caution Budget",
     "risk_tolerance_point1_percent_explanation": "Budget: 0.1% chance of COVID per year (20 microCOVIDs per week) (suggested if you or your close contacts are more vulnerable to COVID)",
     "intro": {
-      "whats_this": "We’ve constructed a calculator that lets you estimate the risk of getting COVID from a wide range of activities, using the <1>best research available</1>. We hope this tool will help hone your intuition, lower your stress levels, and figure out good harm-reduction strategies.",
-      "call_to_action": "Calculate the approximate COVID risk of any activity"
+      "whats_this": "We’ve constructed a calculator that lets you estimate the risk of getting COVID from a specific activity or relationship in your daily life, using the <1>best research available</1>. We hope this tool will help hone your intuition, lower your stress levels, and figure out good harm-reduction strategies.",
+      "call_to_action": "Calculate the COVID risk to you from a specific activity or relationship"
     },
-    "risk_step_label": "Step 2: Describe the activity",
-    "type_of_interaction": "Is this a single activity or an ongoing relationship?",
+    "risk_step_label": "Step 2: Describe the activity or relationship",
+    "type_of_interaction": "Is this a specific activity or an ongoing relationship?",
```

Comment:
Do you have a better idea of how to help users like this, who thought the calculator was for "risk to me in general", not "risk to me from something specific I do"? https://twitter.com/Geo_Rex_H/status/1328135514935812097

Reply:
(apologies this was a more tacit part of the PR description)

Reply:
Work is not an "ongoing relationship", but it is a specific thing I do even though it is not a "one-time" activity

Reply:
"Is the thing we actually want to ask "do you spend more than 5 hours a week indoors with these people?" -> No, because it's not 5 hrs if masked, or ventilated, or ....  it's too confusing to try to explain when to "change over" to using the cap. I don't want to get into that in this PR.

Reply:
I don't think this is going to improve that user's experience; changing "single" to "specific" doesn't help them figure out their total risk. If we want to help them, we need to explain that you need to enter all your activities and relationships for the last 9 days and add them up - I don't think we ever say that.

Reply:
Maybe run it by disc-usability-and-confusion for another opinion?

Reply:
I just meant "I assumed it was going to tell me what risk I (and my wife) were at in general. I was confused when it asked for specific incidents". So he wouldn't assume and wouldn't be confused. 

I did not mean "how do I add up risks to see how risky my behaviour is?". Obviously this does not help with that.

Reply:
Hmmm. I don't have a strong opinion here. I don't think changing this one word (or keeping it) will really make that big of a difference. I think better to just get it out. Not sure how helpful that is. But there ya have it.

Reply:
So let me get this straight:

User: "I was confused when it asked for specific incidents"
Us: We we will fix this by changing the wording to ask for "specific incidents"

It seems to me that the problem is that the user wants the calculator to be omniscient and magically know his risk. If we want to address that confusion, we'd need to know why he thought that. Are we advertising the calculator to do this? Is someone else saying it does? Or is this just a case of someone projecting their wants onto our product?

Anyway, I'd prefer it were "single" instead of "specific" but won't hold up the change over that word.

Reply:
In my mind it's like

User: "When you said burger, I assumed beef burger. I was confused when you gave me a lamb burger"
Us: We will fix this by changing the wording to say it is a "lamb burger".

I think the user was projecting their wants onto the product.